### PR TITLE
Hardhatify Poll, PollCreator, VariableSupplyToken, ServiceRegistry tests👷 

### PR DESCRIPTION
**Pull request by kyriediculous**
**What does this pull request do? Explain your changes. (required)**
Hardhatify Poll, PollCreator, VariableSupplyToken tests 👷

### CODE REVIEW

File: test/unit/VariableSupplyToken.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+import chai, {expect, assert} from "chai"
+import {solidity} from "ethereum-waffle"
+chai.use(solidity)
 
 describe("VariableSupplyToken", accounts => {
     let token
 
     before(async () => {
-        token = await VariableSupplyToken.new()
+        token = await (await ethers.getContractFactory("VariableSupplyToken")).new()
```
Comment (kautukkundan) [inline test/unit/VariableSupplyToken.js]:
```suggestion
        token = await (await ethers.getContractFactory("VariableSupplyToken")).deploy()
```

File: test/unit/VariableSupplyToken.js [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
     })
 
     describe("burn", () => {
         it("should reduce the supply and balance of the sender", async () => {
-            await token.mint(accounts[0], 500, {from: accounts[0]})
+            await token.mint(accounts[0], 500)
```
Comment (kautukkundan) [inline test/unit/VariableSupplyToken.js]:
```suggestion
            await token.mint(accounts[0].address, 500)
```

File: test/unit/VariableSupplyToken.js [hunk 3]
```diff
@@ -1,28 +1,29 @@
-import expectThrow from "../helpers/expectThrow"
-
-const VariableSupplyToken = artifacts.require("VariableSupplyToken")
+import {ethers} from "hardhat"
+import chai, {expect, assert} from "chai"
+import {solidity} from "ethereum-waffle"
+chai.use(solidity)
 
 describe("VariableSupplyToken", accounts => {
     let token
```
Comment (kautukkundan) [inline test/unit/VariableSupplyToken.js]: should add an `accounts` variable to store `ethers.getSigners()`

File: test/unit/VariableSupplyToken.js [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
 
     describe("burn", () => {
         it("should reduce the supply and balance of the sender", async () => {
-            await token.mint(accounts[0], 500, {from: accounts[0]})
+            await token.mint(accounts[0], 500)
 
-            await token.burn(200, {from: accounts[0]})
+            await token.burn(200)
 
             const balance = await token.balanceOf(accounts[0])
```
Comment (kautukkundan) [inline test/unit/VariableSupplyToken.js]:
```suggestion
            const balance = await token.balanceOf(accounts[0].address)
```

**Review by kautukkundan — CHANGES_REQUESTED** [review]
Test failing for `VariableSupplyToken` due to unavailable signer

**Comment by kyriediculous** [comment]
should be fixed

**Review by kautukkundan — APPROVED** [review]

**Pull request merged**